qiita_pet/templates/sitebase.html

```diff
@@ -374,15 +374,17 @@
                 </ul>
               </li>
                 {% set user_level = user.level %}
-                {% if user_level == 'admin' %}
+                {% if user_level in ('admin', 'wet-lab admin') %}
               <li class="dropdown">
                 <a href="#" data-toggle="dropdown" class="dropdown-toggle">Admin<b class="caret"></b></a>
                 <ul class="dropdown-menu">
                   {% if qiita_config.portal == "QIITA" %}
-                  <li><a href="{% raw qiita_config.portal_dir %}/admin/error/">View Errors</a></li>
-                  <li><a href="{% raw qiita_config.portal_dir %}/admin/approval/">View Studies awaiting approval</a></li>
-                  <li><a href="{% raw qiita_config.portal_dir %}/admin/portals/studies/">Edit study portal connections</a></li>
-                  <li><a href="{% raw qiita_config.portal_dir %}/admin/processing_jobs/">Processing Jobs</a></li>
+                    {% if user_level == 'admin' %}
+                      <li><a href="{% raw qiita_config.portal_dir %}/admin/error/">View Errors</a></li>
+                      <li><a href="{% raw qiita_config.portal_dir %}/admin/approval/">View Studies awaiting approval</a></li>
```

Comment:
A really minor point but shouldn't it be either 'View Studies Awaiting Approval' or 'View studies awaiting approval'?

Reply:
Good point. I guess as this is something that only admins (a handful of users) see and nobody has raised an issue ... I'll leave it as is.